data-report-commons/src/test/java/commons/db/GraphStoreProtocolConnectionTest.java

```diff
@@ -64,10 +75,42 @@ void shouldLogConnection() {
     @Test
     void shouldDelete() {
         var triple = "<https://example.org/a> <https://example.org/b> <https://example.org/c> .";
-        dbConnection.write(triple, Lang.NTRIPLES);
-        dbConnection.delete();
-        var query = QueryFactory.create("SELECT * WHERE { ?a ?b ?c }");
-        var result = dbConnection.getResult(query, new TestFormatter());
-        assertNull(result);
+        dbConnection.write(GRAPH, triple, Lang.NTRIPLES);
+        assertNotNull(dbConnection.fetch(GRAPH));
+        dbConnection.delete(GRAPH);
+        assertThatTheGraphDoesNotExist(GRAPH);
+    }
+
+    private static class TestFormatter implements ResponseFormatter {
```

Comment:
Can you use [TestFormatter.java](https://github.com/BIBSYSDEV/nva-data-report-api/blob/main/testing-utils/src/main/java/no/sikt/nva/data/report/testing/utils/TestFormatter.java) (from testing-utils) here?